src/components/SpiceList.js

```diff
@@ -21,10 +75,10 @@ class SpiceList extends React.Component {
           <div className="filters">
             <div>
               <label>Search: </label>
-              <input type="text" placeholder="Search By Tasting Notes..." />
+              <input type="text" placeholder="Search By Tasting Notes..." onKeyUp={ this.handleSearch } />
```

Comment:
When you're working with input fields in React, the recommended approach is using a [controlled component](https://reactjs.org/docs/forms.html#controlled-components), and setting the input's value to a key in state and using the onChange event handler to update it, like this:

```jsx
<input type="text" placeholder="Search By Tasting Notes..." value={this.state.search} onChange={ this.handleSearch } />
```

We'll cover controlled components more in the lecture tomorrow!